Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Controllers/MSTSBrakeController.cs

```diff
@@ -104,6 +106,17 @@ public override void UpdatePressure(ref float pressureBar, float elapsedClockSec
                     var type = notch.Type;
                     if (OverchargeButtonPressed()) type = ControllerState.Overcharge;
                     else if (QuickReleaseButtonPressed()) type = ControllerState.FullQuickRelease;
+
+                    if (type == ControllerState.Hold || type == ControllerState.Lap || type == ControllerState.MinimalReduction)
+                    {
+                        if (EnforceMinimalReduction)
+                            DecreasePressure(ref pressureBar, MaxPressureBar() - MinReductionBar(), ApplyRateBarpS(), elapsedClockSeconds);
+                    }
+                    else
+                    {
+                        EnforceMinimalReduction = false;
+                    }
```

Comment:
```suggestion
                    switch (type)
                    {
                        case ControllerState.Hold:
                        case ControllerState.Lap:
                        case ControllerState.MinimalReduction:
                            if (EnforceMinimalReduction)
                                DecreasePressure(ref pressureBar, MaxPressureBar() - MinReductionBar(), ApplyRateBarpS(), elapsedClockSeconds);
                            break;
                        default:
                            EnforceMinimalReduction = false;
                            break;
                    }
```
With three or more cases for the same variable, I suggest using a switch-case.